lib/client-assets.php

```diff
@@ -149,6 +149,14 @@ function gutenberg_override_translation_file( $file, $handle ) {
 }
 add_filter( 'load_script_translation_file', 'gutenberg_override_translation_file', 10, 2 );
 
+function gutenberg_override_posttype_labels( $labels ) {
+	$labels->featured_image = 'Feaaatured image';
```

Comment:
looks like the macbook keyboard is not working properly :)

Reply:
Oops :D